mypy.ini

```diff
@@ -0,0 +1,9 @@
+[mypy]
+ignore_missing_imports = True
+strict_optional = False
```

Comment:
Why?

Reply:
It is necessary for delayed initialisation of parameters. Like `param: Optional[int] = None`. Otherwise we must check the parameter's type every time before the usage.